README.md

```diff
@@ -165,6 +161,11 @@ git reset --hard vrms/master
    - `touch vrms/backend/.env`
    - `touch vrms/client/.env`
    - `touch vrms/client-mvp-04/.env`
+    
+   Note 1: In the above example you are trying to create an empty file called `.env` in each of the listed directories: backend, client and client-mvp-04. You can use either `touch <path-to-directory> .env` or navigate to the directory and use `touch .env`
+   
+   Note 2: `touch` is a Unix/Linux or Mac command; It is not available in Windows. In Windows navigate to each of the directories and create a new file called .env
```

Comment:
If you create a new file in the windows explorer , it will create a .txt file. Instead, make the file in any text editor (ie Notepad) and save as a .env

Reply:
@AmyMendelsohn -> I made the change to the text as you requested.  Can you please look and let me know if that reads correctly. (I don't have a Windows machine to test on anymore).